ngraph/test/runtime/pass/opset0_downgrade.cpp

```diff
@@ -509,7 +444,7 @@ namespace
         }
 
         const auto replacement_node =
-            std::make_shared<op::v0::Divide>(sum_node, count_node, op::AutoBroadcastSpec::NUMPY);
```

Comment:
An idea of this pass it's to represent an operation using OPs from opset0 only. This change doesn't make sense in this context. If v0 Divide was removed, all the op_cast have to be removed also.